.github/workflows/new_workflow.yml

```diff
@@ -0,0 +1,63 @@
+# Note: until permissions can be minimized, IAM user credentials require
+#       manual modification to run the code below. 
+
+name: CI
+on:
+  push:
+    branches: [ "alex" ]
+    paths:
+      - globalrx/dle/**
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: pull repo
+        uses: actions/checkout@v3
+ 
+      - name: Configure AWS credentials
+        id: creds
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+        continue-on-error: true
+
+
+      - name: Push cfn to S3
+        run: |
+          aws s3 sync ./dle/cfn s3://globalrx
```

Comment:
Bucket name needs to be globally unique, should be parameterized? Bucket probably needs to be created by Cloudfront, prefix or suffix a random string so multiple deploys to different environments work.